src/components/Memoroom.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+const MemoRoomContainer = styled.div`
+  display: flex;
+  width: 300px;
+  height: 300px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  border: 1px solid black;
```

Comment:
```suggestion
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  align-items: center;
  width: 300px;
  height: 300px;
  border: 1px solid black;
```
해당 css 컨벤션 형식으로 수정하시는건 어떠신가요??

Reply:
네! 수정해볼게요!